modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -28,33 +74,41 @@ metadata:
   name: basic
   namespace: istio-system
 spec:
-# ...
+  addons:
+    grafana:
+      enabled: false
+    kiali:
+      enabled: true
+    prometheus:
+      enabled: true
   meshConfig:
     extensionProviders:
-      - name: tempo
-        zipkin:
-          service: tempo-sample-distributor.tracing-system.svc.cluster.local
-          port: 9411
-  tracing:
-    sampling: 10000
-    type: None <1>
-  version: v2.5
----
-kind: ServiceMeshMemberRoll
-apiVersion: maistra.io/v1
+      - name: otel
+        opentelemetry:
+          port: 4317
+          service: otel-collector.bookinfo.svc.cluster.local
```

Comment:
This is a collector for a particular namespace, bookinfo in this case, right? 
So do I need to have `OpenTelemetryCollector` in each service mesh member namespace, and so, do I need to insert all of them here into smcp as well?

Reply:
```
do I need to insert all of them here into smcp as well? ( so for each SMMR where I want to grab tracing spans, I need the extension provider specified here, right? )
```
probably not, since the name needs to be `otel`, otherwise it doesn't work. I also noticed that I see traces from all my mesh applications, not only from Bookinfo. So, why the opentelemetrycollector is in the bookinfo namespace and not in the servicemesh namespace namespace if it collects traces from all mesh namespaces? ( it is a little bit confusing now, because, it seems the each application namespaces need a separate opentelemetrycollector)

Reply:
I answered this in our slack thread.
We only need one instance of the otel collector in one SMMR member namespace such as bookinfo.

We can add a note right after the example yaml manifest.